frontend/src/Components/Login/Login.jsx

```diff
@@ -1,6 +1,117 @@
-import React from "react";
+import React, { useEffect, useRef, useState } from "react";
 import "./Login.css";
+import logoImage from "./assets/logoblackandwhite.svg";
+import AxiosInstance from "../axios/AxiosInstance";
 
-export const Login = () => {
-  return <div>SkyNest</div>;
+const Login = ({ setAccessToken }) => {
+	const [email, setEmail] = useState("");
+	const [password, setPassword] = useState("");
+	const [errorMsg, setErrorMsg] = useState("");
+
+	const emailRef = useRef();
+
+	const getUserToken = async () => {
+		AxiosInstance.post("/login", {
+			email: email,
+			password: password,
+		})
+			.then(({ data }) => {
+				setAccessToken(data.accessToken);
```

Comment:
What if you do not get `data` property for any reason from BE? This will break. Make sure to check those stuff to be safe on FE. Same for `response` on catch.

Reply:
Also, try async/await syntax, for me it's cleaner.

Reply:
Yeah it will , i totally missed that, will fix it, thanks :)